packages/admin-ui/src/stack/stack.css.ts

```diff
@@ -10,26 +10,35 @@ export const stackStyle = (
 })
 
 export const stackTheme = csx({
-  display: 'flex',
+  display: 'inline-flex',
```

Comment:
This fixes the problem with the size of the element when the direction is set to "row"

**Before**
<img width="1209" alt="Captura de Tela 2023-03-17 às 10 53 30" src="https://user-images.githubusercontent.com/20579226/225924666-7e25012f-4c0c-42e3-9cac-5af48eae9807.png">

**After**
<img width="217" alt="Captura de Tela 2023-03-17 às 10 51 12" src="https://user-images.githubusercontent.com/20579226/225924089-0f8dfbd6-4fd4-4d1b-948d-18e39fd55e00.png">